secp256k1-sys/depend/secp256k1/src/secp256k1.c

```diff
@@ -8,6 +8,7 @@
 
 #include "../include/secp256k1.h"
 #include "../include/secp256k1_preallocated.h"
+#include "../../../wasm-sysroot/wasm.h"
```

Comment:
You should likely patch this by means of a `.patch` file that is automatically applied by `vendor.sh` otherwise this change will be gone with the next update of `libsecp256k1`.

Reply:
oh coool, that will fix my unease mentioned in the commit message about patching vendored code :)  Thanks man.

Reply:
In this case you can also append `-include ../../../wasm-sysroot/wasm.h` to the command line of the C compiler (https://docs.rs/cc/latest/cc/struct.Build.html#method.flag), this works on gcc and clang at least.